packages/fxa-auth-client/lib/client.ts

```diff
@@ -1231,12 +1231,14 @@ export default class AuthClient {
     sessionToken: hexstring,
     recoveryKeyId: string,
     recoveryData: any,
-    enabled: boolean
+    enabled: boolean,
+    replaceKey: boolean
```

Comment:
This is technically a 'breaking change', and for shared libraries or public APIs its good to avoid breaking changes if possible. 

I noticed that in settings you have a default of false for replaceKey. Would doing the same thing here be possible? Another reason to provide a default is due to the fact this code is technically a shared library, so it's possible this function gets called from javascript code that does not enforce type safety.

Reply:
I've added defaults for both `enabled` and `replaceKey`, as both could be undefined.